elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -603,12 +604,19 @@ public RelationshipType getRelationshipType(Object entity, String relation) {
 
     /**
      * Get a type for a field on an entity.
+     * <p>
+     * If the field is "id" and the ID name of the identity is not "id", this method returns the type of the field that
+     * is annotated by {@link @Id}.
```

Comment:
I think your comments are breaking the javadoc generation.  My guess is the second `@Id`.  Travis build fails for this PR.

Reply:
Thank you! Let me remove it